README.md

```diff
@@ -26,6 +26,9 @@ the JFR datasource for Grafana.
 for the Grafana dashboard.
 
 # Using
+
+Minimum Supported Kubernetes Version: `v1.21`.
+
```

Comment:
Maybe this should have a requirements list like what is in `Building`.

```markdown
# Using

## Requirements

- `kubernetes` v1.21+
- `operator-lifecycle-manager`
- `cert-manager` (optional)

# Building

## Requirements

- `go` v1.21+
- `operator-sdk` v1.31.0
- `podman` (or `buildah`?) or `docker`
- `jq` v1.6+
- `ginkgo` (optional)
- `cert-manager` (optional)

Reply:
Good idea, updated now :D